tools/proofers/spellcheck_text.inc

```diff
@@ -494,9 +509,9 @@ function output_wordcheck_interface($user, $ppage, $text_data, $is_changed, $acc
     // text highlight legend
     if ($highlights) {
         echo "<p>";
-        echo _("The following highlights have been used:") . "<br>";
+        echo _("The following highlights have been used:") . $spacer;
```

Comment:
Just checking it was intentional to change the spacer from br to space for the list of highlights as well as the list of languages when viewed in the horizontal interface - it looks good on-screen

Reply:
Yes, this was an intentional change for the horizontal interface for the highlight output.